.github/workflows/ccpp.yml

```diff
@@ -84,3 +84,45 @@ jobs:
             else \
               exit 0; \
             fi
+
+  build-and-check-older-gcc:
+
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Deps
+      run: |
+          sudo apt-get update;
+          sudo apt-get install -y \
+                  automake \
+                  autoconf \
+                  libtool \
+                  autogen \
+                  bison \
+                  flex \
+                  libgmp3-dev \
+                  libmpfr-dev \
+                  libmpc-dev \
+                  build-essential \
+                  gcc-4.8 \
+                  g++-4.8 \
+                  gcc-4.8-multilib \
+                  g++-4.8-multilib \
+                  dejagnu
+
+    - name: Configure
+      run: |
+           mkdir -p gccrs-build;
+           cd gccrs-build;
+           ../configure \
+               CC='gcc-4.8' \
+               CXX='g++-4.8' \
+               --enable-languages=rust \
+               --disable-bootstrap \
+               --enable-multilib
+
+    - name: Build
+      shell: bash
+      run: |
+           make -C gccrs-build -j $(nproc)
```

Comment:
lets run the testsuite as well, but we don't need to exit on any unexpected failures

Reply:
Why not check the results?  This is quick, and should never ever differ from the main *build-and-check* job -- and if it ever does, there's a big problem that we'd like to learn about.

Reply:
Actually yeah good point if we ignore any failures we won't notice anything good point. Yeah lets copy this over from the normal job.

Reply:
Done in [74078a6](https://github.com/Rust-GCC/gccrs/pull/1059/commits/74078a6242f73bb3c11d2985f71c30e5f2d1eb72)